[IMP] Livechat: update participate (#12983)
odoo/documentation: content/applications/websites/livechat/participate.rst

that is designated on their profile page.

.. example::

>    Mitchell Admin has his full name as his :guilabel:`User Name`, but he does not want to include

Reviewer: optional rewording suggestion (: it works either way!
```suggestion
   Mitchell Admin's full name is also his :guilabel:`User Name`, but he does not want to include
```

   his last name in a live chat conversation for privacy reasons. He would then set his

>    :guilabel:`Online Chat Name` to include only his first name, Mitchell.

Reviewer: great example!

Manage live chat requests
-------------------------

When an operator is active in a channel, chat windows open in the bottom-right corner of the screen,

> no matter where they are in the database. This allows them to access other pages and apps, while

Reviewer 2: ```suggestion
no matter what page they are on in Odoo. This allows them to access other pages and apps, while
```
Wildly optional, and I know this is from the old doc, it just took me a while to process what "where you are" meant specifically.
